### ISSUE TITLE
Field arguments other than limit and offset

### ISSUE THREAD

**Issue by andrew-web** [semantic-integration/hypergraphql#16]
Does HyperGraphQL support field arguments other than `limit` and `offset`? 

Specifically, I would like to query for a specific URI. For the default DBpedia example, the documentation contains an example like this:

```
{
  people(limit: 1, offset: 5) {
    _id
    _type
    name
    birthDate
  }
}
```

and returns `http://dbpedia.org/resource/Alaeddine_Yahia`. I would like to query for this resource directly with a GraphQL query like:

```
{
  people(_id:"http://dbpedia.org/resource/Alaeddine_Yahia") {
    _id
    _type
    name
    birthDate
  }
}
```

or

```
{
  people(name:"Alaeddine Yahia") {
    _id
    _type
    name
    birthDate
  }
}
```

Are these examples possible? If so, is there a simple example that we could show in the documentation? I believe this would benefit others.

**Comment by sklarman** [comment]
Hi Andrew,

We've just released a new version [HyperGraphQL-v0.2](https://github.com/semantic-integration/hypergraphql/releases/tag/0.2), which adds support for exactly such queries. More specifically, whenever you define a type in your schema, e.g. `Person` and associate it with some service, HyperGraphQL will automatically expose two query fields:
- `Person_GET (limit:Int, offset:Int)`
- `Person_GET_BY_ID (uris: [String])`
The first one corresponds to the original types of queries. The second one is there to support exactly the functionality you describe. Take a look at our new documentation and demo website at [http://hypergraphql.org](http://hypergraphql.org).

Please note, that the current release is not backward compatible with the earlier one, with which you're currently working. Sorry for that, but this was unavoidable given that the early one was a very premature prototype.

**Comment by andrew-web** [comment]
Thank you! I have tested the new version and found that GET_BY_ID solves my issue.
